build_tools/buildkite/cmake/build_configurations.yml

```diff
@@ -65,3 +65,8 @@ steps:
       IREE_DOCKER_WORKDIR: "/usr/src/github/iree"
     agents:
       - "queue=build"
+
+# Notify rotation on postsubmit failures.
+notify:
```

Comment:
I'm also looking into alerts/notifications about GitHub Actions (mainly for the release and website publish workflows in https://github.com/google/iree/tree/main/.github/workflows), but there doesn't appear to be any native support for them??

https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/about-notifications is for personal notifications and does nothing with actions running in shared repositories like this.

There's a blog post recommending a "send email" action with all sorts of email server / username / password / subject / body / etc. details: https://www.ravsam.in/blog/send-email-notification-when-github-action-fails/